Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:14 as builder
+
+WORKDIR /app
+
+COPY package*.json ./
```

Comment:
This will copy package-lock.json.
I guess we only need a package.json file and then npm run install in the image
For now, let's do that.